dotnet/src/webdriver/DevTools/DevToolsSession.cs

```diff
@@ -262,12 +268,6 @@ public async Task<JToken> SendCommand(string commandName, string sessionId, JTok
                 millisecondsTimeout = Convert.ToInt32(CommandTimeout.TotalMilliseconds);
             }
 
-            if (this.attachedTargetId == null)
```

Comment:
To be on the very safe side and avoid breaking changes I could also add this removed code again.

Reply:
I propose to remove it here, with assumption that the session  is re-initialized earlier. Given that this method takes `sessionId` as is, so caller should be aware.